docs/udp.rst

```diff
@@ -335,70 +518,48 @@ Another developer already prepared it for you,
 or you prefer to fine-tune process graphs in a JSON editor.
 It is very straightforward to submit this as a user-defined process.
 
-Say we start from the following Python dictionary,
-representing the Fahrenheit to Celsius conversion we discussed before::
 
-    fahrenheit_to_celsius = {
-        "subtract1": {
-            "process_id": "subtract",
-            "arguments": {"x": {"from_parameter": "f"}, "y": 32}
-        },
-        "divide1": {
-            "process_id": "divide",
-            "arguments": {"x": {"from_node": "subtract1"}, "y": 1.8},
-            "result": True
-        }}
+Say we start from the above Python dictionary saved as JSON,
+then you can simply use it using the following code and specifying the 
+needed parameters:
 
-We can store this directly, taking into account that we have to define
-a parameter named ``f`` corresponding with the ``{"from_parameter": "f"}`` argument
-from the dictionary above::
+.. code-block:: python
 
-    connection.save_user_defined_process(
-        user_defined_process_id="fahrenheit_to_celsius",
-        process_graph=fahrenheit_to_celsius,
-        parameters=[Parameter.number(name="f", description="Degrees Fahrenheit")]
+    shared_graph = connection.datacube_from_json(
+        'shared_process_graph.json',
+        parameters={
+            "temporal_interval": date,
```

Comment:
`date` is confusing here as it looks like a single date, but you have to pass a two-item list/tuple here right?